### DISCUSSION TITLE
Better augmentations

### DISCUSSION THREAD

**Discussion by ddbuscombe** [Doodleverse/segmentation_gym#60] [General]
abandon imagedatagenerator in favor of a better, faster set of augmentation tools that allow for spectral transformations and custom transformations

https://github.com/albumentations-team/albumentations

**Comment by ddbuscombe** [comment]
https://sayak.dev/tf.keras/data_augmentation/image/2020/05/10/augmemtation-recipes.html

**Comment by ebgoldstein** [comment]
https://github.com/facebookresearch/AugLy

**Comment by ddbuscombe** [comment]
https://kornia.github.io/
